unit/cq_test.c

```diff
@@ -89,6 +127,7 @@ cq_open_close()
 			FT_WARN("\nSuccessfully completed %d iterations up to size %d "
 					"before the provider returned EINVAL...",
 					i + 1, size >> 1);
+			maximum_size = size >> 1;
```

Comment:
This can be moved up and FT_WARN can use maximum_size instead of calculating (size >> 1).